tool/tsh/tsh.go

```diff
@@ -611,6 +611,14 @@ func onLogout(cf *CLIConf) {
 	// Extract all clusters the user is currently logged into.
 	active, available, err := client.Status("", "")
 	if err != nil {
+		if trace.IsNotFound(err) {
+			fmt.Printf("All users logged out.\n")
+			return
+		} else if trace.IsAccessDenied(err) {
+			fmt.Printf(err.Error())
+			fmt.Printf(": Logged in user does not have the correct permissions. \n")
```

Comment:
nit: combine these lines: `fmt.Printf("%v: Logged in user does not have the correct permissions\n", err)`